scripts/dasharo-deploy

```diff
@@ -372,15 +381,21 @@ install_ec() {
 
 install() {
   ask_for_version
-  download_artifacts
-  display_flashing_warning
-  check_flash_lock
-  verify_artifacts bios
+  if [ "$BOARD_VERSION" == "V540TU" ] || [ "$BOARD_VERSION" == "V560TU" ]; then
```

Comment:
Adding board-specific flags here in common code may be tricky in the long run.
I suspect we will have more of such models, even rather soon, e.g. the same variants with nvidia @mkopec  ?  Then we will have 4 boards here, maybe 8 in while?

Maybe add some flag indicating only EC flashing is supported, and flag is set in the board_config() ?

Reply:
But having this option of flashing EC with one click (instead of downloading EC firmware manually and flashing manually) will be nice, I hope @wessel-novacustom will find it useful :blush:

Reply:
@macpijan Very useful. We now flash EC with an own script where we ask if the platform is V540TU or V560TU. If this can somehow be automatically recognised, that would be helpful too.

Reply:
The final release firmware will have SMBIOS Baseboard Version set to V540TU or V560TU depending on the exact model. The release candidates (before rc10) don't have that yet, but EC model can also be read from DTS using `dasharo_ectool info`